src/Microsoft.DotNet.Cli.Utils/FrameworkDependencyFile.cs

```diff
@@ -45,6 +46,36 @@ public string GetNetStandardLibraryVersion()
                 ?.Version;
         }
 
+        public bool TryGetMostFitRuntimeIdentifier(IEnumerable<string> candidateRuntimeIdentifiers, out string mostFitRuntimeIdentifier)
+        {
+            mostFitRuntimeIdentifier = null;
+
+            if (!SupportsCurrentRuntime())
+            {
+                return false;
+            }
+
+            RuntimeFallbacks runtimeFallbacks
+                = DependencyContext.RuntimeGraph.Where(g => g.Runtime == RuntimeEnvironment.GetRuntimeIdentifier()).Single();
```

Comment:
You shouldn't call `RuntimeEnvironment.GetRuntimeIdentifier()` at every iteration of this loop.  Pull that call out into a variable.